engine/runtime-integration-tests/src/test/java/org/enso/interpreter/node/expression/builtin/meta/TypeOfNodeTest.java

```diff
@@ -56,7 +57,7 @@ public static Object[][] allPossibleEnsoInterpreterValues() throws Exception {
         data.add(new Object[] {raw, n});
       }
     }
-    data.add(new Object[] {UnresolvedSymbol.build("unknown_name", null), "Function"});
+    data.add(new Object[] {UnresolvedSymbol.build("unknown_name", (ModuleScope) null), "Function"});
```

Comment:
Why is this change necessary? Casting `null` to `ModuleScope`?

Reply:
`build` is currently overloaded.